src/valueObjects/Cursor.js

```diff
@@ -0,0 +1,52 @@
+import isArray from "lodash/isArray";
```

Comment:
Should this file be moved/renamed? `Cursor` is not actually a class, just a definition and validation function for a specified set of properties on any object.

Reply:
Any reason not to provide a Cursor factory?